src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -31,7 +31,6 @@ internal static class CodeActionHelpers
         /// </remarks>
         public static async Task<VSInternalCodeAction[]> GetVSCodeActionsAsync(
             CodeActionParams request,
-            CodeActionsCache codeActionsCache,
```

Comment:
The code actions cache was never actually read from, it was only updated.  So I removed it for now as I wasn't quite sure how it was supposed to work.

Will need to revisit it if we decide we need it.